envoyauth/response_test.go

```diff
@@ -455,6 +455,127 @@ func TestGetResponseHeaders(t *testing.T) {
 	if len(result) != 2 {
 		t.Fatalf("Expected two header but got %v", len(result))
 	}
+	//
```

Comment:
Commented code to be removed?

Reply:
Mistake, will uncomment it. Introduced it to valid the case of `[]interface{}` for header values.